changelogs/fragments/0-postgresql_user.yml

```diff
@@ -0,0 +1,2 @@
+bugfixes:
+- postgresql_user - didn't close connection to DB. Fix now connection to DB is closed `` (https://github.com/ansible-collections/community.postgresql/issues/431).
```

Comment:
```suggestion
- postgresql_user - Properly close DB connections to prevent possible connection limit exhaustion (https://github.com/ansible-collections/community.postgresql/issues/431).
```